src/main/app/cypress/integration/komo-aikuisten-perusopetus.spec.js

```diff
@@ -0,0 +1,37 @@
+import { playMocks } from 'kto-ui-common/cypress/mockUtils';
+
+import komoTuvaMocks from '#/cypress/mocks/komo-aikuisten-perusopetus.mocks.json';
+
+describe('Aikuisten perusopetus KOMO', () => {
+  beforeEach(() => {
+    playMocks(komoTuvaMocks);
+    cy.visit('/fi/koulutus/1.2.246.562.13.00000000000000002339');
+
+    // Wait for everything to load
+    cy.findByRole('progressbar').should('not.exist');
+  });
+
+  it('renders perustiedot with koulutustyyppi and opintojenlaajuus', () => {
+    cy.get('h1').contains('Aikuisten perusopetus');
+    cy.findByTestId('koulutustyyppi').contains('Aikuisten perusopetus');
+    cy.findByTestId('opintojenLaajuus').contains('18 viikkoa');
```

Comment:
En ole näitä testimuutoksia kokeillut, mutta tähän tyyliin näiden pitäs toimia. Testi-id:itä kannattaa käyttää säästeliäästi, koska niiden avulla tehdyt queryt ei vastaa sitä miten käyttäjä löytää tietyt elementit sivulta.

```suggestion
    cy.findByLabelText('Koulutustyyppi').should('have.text', 'Aikuisten perusopetus');
    cy.findByLabelText('Koulutuksen laajuus').should('have.text', '18 viikkoa');
```